db/migrate/20171113145621_add_resource_type_and_resource_id_to_resource_monitor.rb

```diff
@@ -1,6 +1,5 @@
 class AddResourceTypeAndResourceIdToResourceMonitor < ActiveRecord::Migration[5.0]
   def change
-    add_column :resource_monitors, :resource_type, :string
-    add_column :resource_monitors, :resource_id, :integer, index: true
+    add_column :resource_monitors, :reference, :string, null: false, index: true
```

Comment:
I think you'll need to put this into a separate migration since this one will have already run on integration.